benchmarks/models.py

```diff
@@ -231,3 +292,132 @@ class MailingList(models.Model):
     indexes = [
         models.Index(fields=['email']),
     ]
+
+class JSONBField(models.JSONField):
+    """
+    Django's standard JSONField tries to decode JSON "strings" into dicts/lists.
+    However, in the materialized view creation, JSONB instead of JSON is used.
+    Psycopg2 automatically converts JSONB to python dicts/lists.
+
+    JSONField led to a TypeError because it would receive a string instead of
+    an already decoded Python object.
+
+    JSONB is also more performant for large datasets (like FinalModelContext)
+    """
+    def from_db_value(self, value, expression, connection):
+        # 1) If DB returned None, just pass it
+        if value is None:
+            return None
+
+        # 2) If DB gave us dict/list, return as is
+        if isinstance(value, (dict, list)):
+            return value
+
+        # 3) Otherwise assume it's a JSON string, parse with json.loads
+        return json.loads(value)
+
+class FinalBenchmarkContext(models.Model):
```

Comment:
is there a way to auto-fill this from the models this is pieced together? seems redundant otherwise